### ISSUE TITLE
examples _and_ exemples?

### ISSUE THREAD

**Issue by bmorphism** [kauemurakami/getx_pattern#41]
[as of writing](https://github.com/kauemurakami/getx_pattern/tree/8ab536d37195a32665694434e1b3cf2ddd90fde7), the repo has `examples` and `exemples`, and both have stuff in them - is this intentional (for existing tutorials)?

**meta (optional):** do you think `getx_pattern` is going to make it? and has a different templating approach emerged?
I am looking to put together a set of starter patterns for various "web3" 🦍 use-cases, and came across this after `get` seemed like the most compatible approach to state management. If you were starting today, what would you do differently?

**Comment by kauemurakami** [comment]
Goodnight @bmorphism , sorry for the delay in this answer, which by the way is a great question.

As I gained more experience, and with the package's own evolution, I found some ways to improve some things, but the structure, in a macro way, didn't have many changes.

And an important detail is my desire to transform it into a universal pattern that can be used in several other approaches, initially I validated it and had several other ideas when implementing the same model in javascript with node, I will attach an example of the structure.

![image](https://user-images.githubusercontent.com/16777838/190023969-7078efdb-f143-40e4-af67-83890f11ae2a.png)

So yes, I'm still learning and trying to improve the package, and I should release another one by the end of the year for a global edge like minimals architecture.

As for whether I would do it differently, I don't think so, and I think that the path so far, with all the hits and misses, made me and the package what it is.
Also remembering that I was based on other models such as MVP, MVC and bloc.
Even though it was at the beginning of my career, I felt that the pain that had to be healed was that of the people who came from these technologies, web or other state managers to getx, and still didn't know a way to "fit" any of these existing approaches, and I also wanted to, lol and by serving me I thought I could help other people.

After some time, I managed to implement, even with some other state managers the same model.

So I think I did what I could for this migration and getx initiation goal.

But now, I'm pissed at some extremely massive patterns and a high level of boilerplate in several areas, not just in flutter lol , which made me look in a more macro and adaptive way, like turning the getx pattern into something extremely easy, adaptable and understandable for other sectors like web, backend.

 And that's what I'm currently working on.
